README.md

```diff
@@ -27,6 +27,20 @@ that creates additional scrape targets. Each scrape target is expected
 to expose a `/metrics` HTTP path that exposes its metrics in a
 Prometheus compatible format.
 
+For example if it desired to scrape metrics regarding the state of
+objects in a Kubernetes cluster, the
+[kube-state-metrics](https://charmhub.io/kube-state-metrics) charm may
+be used for this purpose. All that is required, in addition to
+deploying Prometheus as above, is to deploy the `kube-state-metrics`
+charm and add a relation with the Prometheus charm as shown below.
+
+
+    juju deploy kube-state-metrics
```

Comment:
```sh
$ juju deploy kube-state-metrics
# juju relate kube-state-metrics prometheus-k8s
```

Reply:
Not sure what the difference here is, Just the dollar sign ?